db/migrate/20231117072002_create_comments.rb

```diff
@@ -0,0 +1,14 @@
+class CreateComments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Good job so far on this one @paulshaun-kdtk, but there are a few points I would like for you to work on:
- A comment must have only the comment text and only that. It's only a post that has a title and text/post body, while a comment is just the comment text made on a post. Kindly remove the title field for a comment. It's not specified on the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) given. 
- According to the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), we don't have comments_counter and likes_counter for comment but only for a post. Please remove them along with the title field. 👍🏽